config/formats.js

```diff
@@ -541,7 +541,15 @@ let Formats = [
 		},
 		onModifyPriority: function (priority, pokemon, target, move) {
 			// @ts-ignore
-			if (move.category !== 'Status' && pokemon && pokemon.forte) return priority + pokemon.forte.priority;
+			if (move.category !== 'Status' && pokemon && pokemon.forte) {
+				if (pokemon.getAbility().onModifyPriority) {
+					// @ts-ignore
+					priority += pokemon.forte.priority;
+					return this.singleEvent('ModifyPriority', pokemon.getAbility(), pokemon.abilityData, pokemon, target, move, priority);
```

Comment:
```suggestion
					return this.singleEvent('ModifyPriority', pokemon.getAbility(), pokemon.abilityData, pokemon, target, pokemon.forte, priority);
```

It should be passing the forte this time around

Reply:
Idk why i always forget to change that